core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -19,8 +19,14 @@
  * you change or add Alluxio configuration properties.
  */
 public enum PropertyKey {
+  CONF_DIRECTORY(Name.CONF_DIRECTORY, "${alluxio.home}/conf"),
   DEBUG(Name.DEBUG, false),
+  /**
+   * @deprecated use INSTALL_HOME or WORK_DIRECTORY instead
+   */
+  @Deprecated
```

Comment:
about the deprecation:

I still kinda like this property alluxio.home.

another way is to introduce alluxio.conf.directory and alluxio.work.directory, but still keep alluxio.home, and set conf.dir defaults to home/conf, and worker.dir defaults to home?